tests/Feature/Lib/UsersUnitTest.php

```diff
@@ -205,4 +205,67 @@ public function get_email(
 
 		return $response;
 	}
+
+	/**
+	 * Retrieve currentUser.
+	 *
+	 * @param int         $expectedStatusCode
+	 * @param string|null $assertSee
+	 *
+	 * @return TestResponse
+	 */
+	public function get_user(
+		int $expectedStatusCode = 200,
+		?string $assertSee = null
+	): TestResponse {
+		$response = $this->testCase->postJson('/api/User::getCurrent');
+		$response->assertStatus($expectedStatusCode);
+		if ($assertSee) {
```

Comment:
```suggestion
		if ($assertSee !== null) {
```

I know PHPStan is not (yet?) enabled for the tests, but maybe it will someday.